test/compilable/commaexp.d

```diff
@@ -0,0 +1,43 @@
+/* REQUIRED_ARGS: -o-
+TEST_OUTPUT:
+---
+compilable/commaexp.d(19): Deprecation: Using the result of a comma expression is deprecated
+compilable/commaexp.d(36): Deprecation: Using the result of a comma expression is deprecated
+compilable/commaexp.d(37): Deprecation: Using the result of a comma expression is deprecated
+compilable/commaexp.d(38): Deprecation: Using the result of a comma expression is deprecated
+compilable/commaexp.d(39): Deprecation: Using the result of a comma expression is deprecated
+compilable/commaexp.d(41): Deprecation: Using the result of a comma expression is deprecated
+compilable/commaexp.d(42): Deprecation: Using the result of a comma expression is deprecated
+---
+*/
+
+class Entry {}
```

Comment:
That's weird...  Are you sure it wasn't just the one on line 19, being outside the function?